core/src/main/java/com/novoda/noplayer/model/Either.java

```diff
@@ -0,0 +1,50 @@
+package com.novoda.noplayer.model;
```

Comment:
Are we exposing this to clients? If not the project is using the convention of `internal` to describe public classes which do not provide a guaranteed api.

Reply:
Not directly, but it's used in `SurfaceRequester` which is kinda exposed in the public API through `PlayerSurfaceHolder`(the clients need to provide an instance of that when they implement a custom implementation of `PlayerView`).